src/app/static/src/app/components/plots/bar/utils.ts

```diff
@@ -1,14 +1,13 @@
-import {ChartData, ChartDataset} from "chart.js";
-import {BarchartMetadata, IndicatorMetadata, FilterOption} from "../../../generated";
+import {ChartData, ChartDataset, DefaultDataPoint} from "chart.js";
+import {IndicatorMetadata, FilterOption} from "../../../generated";
 import {AnnotationOptions} from "chartjs-plugin-annotation";
 import {formatOutput} from "../utils";
 import {PlotData} from "../../../store/plotData/plotData";
 import {Dict} from "../../../types";
 
-export type BarchartIndicatorMetadata = BarchartMetadata["indicators"][0];
-export interface BarChartData extends ChartData {
-    maxValuePlusError: number;
-}
+type BarChartDataset<Data> = ChartDataset<"bar", Data>
+
+type BarChartDefaultData = DefaultDataPoint<"bar">
```

Comment:
these are both basically just types from `chart-js` but ive specified the generic of `ChartType` to be `"bar"` for use later